i18n-helpers/src/lib.rs

```diff
@@ -315,6 +338,22 @@ pub fn group_events<'a>(events: &'a [(usize, Event<'a>)]) -> Vec<Group<'a>> {
 
                         ctx.skip_next_group = true;
                     }
+
+                    Some(directives::Directive::TranslatorComment(translator_comment)) => {
```

Comment:
We can have a little less "stuttering" by using `comment` to mean `translator_comment` in our local variables:

```suggestion
                    Some(directives::Directive::TranslatorComment(comment)) => {
```

I would also be totally okay with `TranslatorComment` being just `Comment` since I don't think we'll have any other kind of comment?

Reply:
Sounds good!  Dropping the term "translator_comment" everywhere with just "comment".